src/index.js

```diff
@@ -1,36 +1,46 @@
-'use strict'
-
-const core = require('@actions/core')
-const differenceInDays = require('date-fns/differenceInDays')
-const { logInfo, logError } = require('./log')
-const { getLastReleaseDate } = require('./getLastReleaseDate')
-const { getLastRepoUpdate } = require('./getLastRepoUpdate')
-const { createIssue } = require('./createIssue')
+const core = require('@actions/core');
+const { logInfo } = require('./log');
+const { getLatestRelease, getUnreleasedCommits } = require('./release');
+const { createIssue } = require('./utils');
 
 async function run() {
   try {
-    const daysToStaleRelease = core.getInput('days-to-stale-release')
+    logInfo('========Starting to run the stale release github action ============');
 
-    const { lastReleaseDate, npmError, npmPckNotFound } = await getLastReleaseDate()
+    const token = core.getInput('github-token', { required: true });
 
-    if (npmPckNotFound) return logError('npm package not found')
+    const daysToIgnore = core.getInput('days-to-ignore');
```

Comment:
is this required? if not, it should have a default value. see other comment about this too.

Reply:
It takes the input from the user. Also, I have now included a default value in parameters in case the user doesn't provide one

Reply:
why don't we simply make it a required input? or non-required with a default is also fine. in the second case, let's document what is the default, of course

Reply:
I have made it optional with default being 0 (documented in the readme)

Reply:
do you think 0 is a sensible default?